src/options/quantifiers_options.toml

```diff
@@ -1066,12 +1066,12 @@ header = "options/quantifiers_options.h"
   help       = "enable approach which unifies conditional solutions, specialized for programming-by-examples (pbe) conjectures"
 
 [[option]]
-  name       = "sygusPbeMultiFair"
+  name       = "sygusPbeMultiFairDiff"
   category   = "regular"
-  long       = "sygus-pbe-multi-fair"
-  type       = "bool"
-  default    = "true"
-  help       = "when using multiple enumerators, ensure that we only register value of minimial term size"
+  long       = "sygus-pbe-multi-fair-diff=N"
```

Comment:
I'd also like to have an option where N is "infnity", i.e. we don't really enforce any kind of fairness. What do you think?

Reply:
Yeah you're right, I'll add the previous flag and this integer.